example_projects/transfer/nlb_observability_stack/s3.tf

```diff
@@ -0,0 +1,70 @@
+resource "aws_s3_bucket" "nlb_logs" {
+  bucket = var.bucket_name
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "bucket-config" {
+  bucket = aws_s3_bucket.nlb_logs.id
+
+  rule {
+    id = "short_retention_policy"
+
+    expiration {
+      days = 4
+    }
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_public_access_block" "nlb_logs" {
```

Comment:
I suggest to remove `aws_s3_bucket_public_access_block` and `aws_s3_bucket_server_side_encryption_configuration` from this example

It doesn't relate to us

Reply:
removed aws_s3_bucket_public_access_block, left  aws_s3_bucket_server_side_encryption_configuration but made it conditional from variable (left it since recommended in the AWS access logs setup description)